Multisig/Cross-layer/Configuration/Config.Example.xcconfig

```diff
@@ -131,6 +131,8 @@ HELP_CENTER_URL         = https:/$()/help.gnosis-safe.io/
 FEATURE_SUGGESTION_URL  = https:/$()/safe.cnflx.io/
 CONTACT_EMAIL           = safe@gnosis.io
 
+APP_STORE_REVIEW_URL = https:/$()/apps.apple.com/app/id1515759131?action=write-review
```

Comment:
Just wanted to note that using app store id will make this url work only for current app but not others, ie. staging Rinkeby..

you can do same thing using the bundle id using the following url formate 
https://itunes.apple.com/@StoreID/lookup?bundleId=@BundleID

Easy way to get storeID as explained here: 
https://stackoverflow.com/a/61597181/5093995

Reply:
nice catch, will update the setting